pep-0539.txt

```diff
@@ -327,14 +330,15 @@ Rejected Ideas
   Python should not be hobbled on affected platforms despite them being
   otherwise perfectly capable of running multi-threaded Python.
```

Comment:
I'd suggest to add a sentence at the end `This idea cannot be adopted because the ``--without-threads`` option has since been removed [16]_.`. And the footnotes `[16]: https://bugs.python.org/issue31370`.